Hab2.0_Getting_Started.md

```diff
@@ -0,0 +1,306 @@
+
+# Create a minimal test plan
+cat > plan.sh << 'EOF'
+pkg_name=test-service
+pkg_origin=chef-private
+pkg_version="1.0.0"
+pkg_maintainer="Test User <test@company.com>"
+pkg_license=('Apache-2.0')
+pkg_description="Test service for Habitat 2.0 migration validation - Cross Platform"
+
+# Cross-platform dependency
+pkg_deps=(
+  core/python
+)
+
+# Platform-agnostic service using Python HTTP server
+pkg_svc_run="python3 -m http.server 8080"
+
+do_build() {
+  return 0
+}
+
+do_install() {
+  return 0
+}
+EOF
+```
+
+### Build Against Stable Channel
+
+```bash
+# Enter studio with stable channel
+export HAB_BLDR_CHANNEL=stable
+export HAB_REFRESH_CHANNEL=stable
+
+hab studio enter
+
+# Inside studio - build the package
+build
+
+# Exit studio
+exit
+```
+
+### Alternative: Use Existing Component
+
+For more realistic testing, use an existing component like `core/nginx`:
+
+```bash
+# Test with existing core package
+sudo hab pkg install core/nginx
+sudo hab svc load core/nginx
+```
+
+### Verification Steps
+
+- Build completes successfully
+- Package artifact is created in `results/` directory
+- Dependencies are pulled from stable channel (verify in build output)
+
+---
+
+## Step 4: Run the Built Package in Supervisor
+
+### Load and Start the Service
+
+```bash
+# Find the built package
+PACKAGE_PATH=$(find results/ -name "*.hart" -type f | head -1)
+echo "Built package: $PACKAGE_PATH"
+
+# Install the package
+sudo hab pkg install $PACKAGE_PATH
+
+# Load the service in supervisor
+sudo hab svc load chef-private/test-service
+
+# Verify service is loaded
+sudo hab svc status
+```
+
+### Verification Steps
+
+- Service loads without errors
+- Service shows "UP" status in `hab svc status`
+
+---
+
+## Step 5: Setup HAB Auth Token with License Key
+
+### Obtain License Key and Token
+If you dont have one already, generate a free/trial license key to go ahead https://www.chef.io/license-generation-free-trial.
+Once generated, you should have the below info ready with you:
+- Valid license key for Habitat 2.0
+- Auth token associated with the license
+
+### Configure Authentication
+
+```bash
+# Set the auth token
+export HAB_AUTH_TOKEN=your_auth_token_here
+```
+
+### Persistent Configuration
+
+```bash
+# Linux/macOS: Add to shell profile
+echo 'export HAB_AUTH_TOKEN=your_auth_token_here' >> ~/.bashrc
+echo 'export HAB_LICENSE=accept-no-persist' >> ~/.bashrc
+
+# Windows: Set environment variable
+setx HAB_AUTH_TOKEN "your_auth_token_here"
+setx HAB_LICENSE "accept-no-persist"
+```
+
+---
+
+## Step 6: Migrate Environment to Habitat 2.0
+
+### (Optional) Stop Current Services
+
+Note: If your supervisor is running services while executing the migration script, these services will be restarted.
+
+```bash
+# Stop all running services
+sudo hab svc unload chef-private/test-service
+
+# Linux: Stop systemd service
+sudo systemctl stop hab-sup
+
+# Windows: Stop Windows service
+hab pkg exec core/windows-service stop
+```
+
```

Comment:
The migration should stop the services and restart systemd/windows service for them so we can leave the above out.

Reply:
addressed in [a69c33c](https://github.com/habitat-sh/habitat/pull/10020/commits/a69c33c16d8ebcc41fe50e6b42c4af968d496ea1)